src/types/store.ts

```diff
@@ -63,7 +63,6 @@ export type GlobalState = {
         posts: PostsRequestsStatuses;
         teams: TeamsRequestsStatuses;
         users: UsersRequestsStatuses;
-        preferences: PreferencesRequestsStatuses;
```

Comment:
Can you remove the corresponding 

```
export type PreferencesRequestsStatuses = {
    getMyPreferences: RequestStatusType;
    savePreferences: RequestStatusType;
    deletePreferences: RequestStatusType;
};
```
In src/types/requests.ts

Reply:
@sudheerDev  Done :+1:

Reply:
Hi @sudheerDev , please let me know if there are any changes required.

Reply:
@atulya-pandey thanks 👍